src/components/Beers.js

```diff
@@ -0,0 +1,34 @@
+import axios from "axios";
+import { useState } from "react";
+import { useEffect } from "react";
+
+function Beers() {
+
+    const [beers, setBeers] = useState([""])
```

Comment:
it's enough to have empty array as initial state, having an empty string inside makes no difference :)